hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/io/BlockOutputStreamEntry.java

```diff
@@ -36,7 +36,14 @@
 import com.google.common.annotations.VisibleForTesting;
 
 /**
```

Comment:
I have consolidated the changes that can go into master under HDDS-5816. PR for that is up under #2709, I am preparing this PR to be based on those changes, so that we will need to wait until the changes are merged in master, and then merged to the base EC branch though.